recipes/_configure_logs.rb

```diff
@@ -0,0 +1,13 @@
+# coding: UTF-8
+
+# Cookbook Name:: cerner_splunk
+# Recipe:: _configure_logs
+#
+# Configures the system log-local.cfg file
+
+log_local_contents = node['splunk']['logs']
+
+splunk_logs '/opt/splunk/etc/log-local.cfg' do
```

Comment:
I've hard-coded the location of log-local.cfg both here and in the libraries/splunk_logs.rb file. Is that acceptable or is there a way that we would prefer to define this as a relative path?

Reply:
The base path is configurable via `node['splunk']['home']`, prepend that to the relative location for /etc/log-local.cfg.

Reply:
Got it, updated here: https://github.com/cerner/cerner_splunk/pull/191/commits/0d57a4e976bd94177822cafac721537ce55cfa1c